components/gnb/gnb.tsx

```diff
@@ -0,0 +1,87 @@
+import Image from 'next/image';
+import Link from 'next/link';
+
+import NotificationBtn from './notificationBtn';
+import SearchBar from './searchBar';
+
+// 더미데이터
+const alertList = {
+  limit: 5, // 알림 개수
+  items: [
+    {
+      item: {
+        id: '1',
+        createdAt: '2024-04-18T08:00:00Z',
+        result: 'accepted', // 또는 'rejected'
+        shop: {
+          item: {
+            id: '1',
+            name: 'Shop A',
+            imageUrl: 'https://example.com/shop_a_image.jpg',
+          },
+        },
+        notice: {
+          item: {
+            startsAt: '2024-04-18T09:00:00Z',
+            workhour: 8,
+          },
+        },
+      },
+    },
+    {
+      item: {
+        id: '2',
+        createdAt: '2024-04-17T12:00:00Z',
+        result: 'rejected', // 또는 'accepted'
+        shop: {
+          item: {
+            id: '2',
+            name: 'Shop B',
+            imageUrl: 'https://example.com/shop_b_image.jpg',
+          },
+        },
+        notice: {
+          item: {
+            startsAt: '2024-04-17T13:00:00Z',
+            workhour: 6,
+          },
+        },
+      },
+    },
+  ],
+};
+function UserTypeLink({ userType }: { userType: string }) {
+  return userType === 'employee' ? <Link href="/my-profile">내 프로필</Link> : <Link href="/my-store">내 가게</Link>;
+}
```

Comment:
P5: 이전 코드와 한 줄 띄워주시면 가독성이 더 좋을 것 같습니다.

P5: 넘어오는 prop이 하나라서 굳이 필요할까 싶긴 하지만, 다른 파일과 형식을 맞추기 위해, Prop 타입 지정을 따로 분리하는 것도 좋을 것 같습니다.

```suggestion
interface UserTypeProps {
  userType: string;
}

function UserTypeLink({ userType }: UserTypeProps) {
  return userType === 'employee' ? <Link href="/my-profile">내 프로필</Link> : <Link href="/my-store">내 가게</Link>;
}
```